### ISSUE TITLE
I want to know whether the data set is real shot or synthesized.

### ISSUE THREAD

**Issue by whattoshow** [BingyaoHuang/CompenNet#7]
Hello ,I'm training the original dataset, and I want to shot some pics for calibration, so I want to know whether the dataset is real shot or sythesized, and should the image be adjusted to look like the one in the data set?

**Comment by BingyaoHuang** [comment]
@whattoshow The images are real captured. You need to capture your own training data if you want to apply our method to your setup.

**Comment by whattoshow** [comment]
> @whattoshow The images are real captured. You need to capture your own training data if you want to apply our method to your setup.

Thank you! I'll capture some images and train them with your mathod!~

**Comment by whattoshow** [comment]
@BingyaoHuang Hi, these days I tried to make a dataset with my own pro-cam system,  I saw the images from cam/warp/train folder look like the surface is fixed and unique, but the reference textured surface folder contains 126 images, I feel a little bit confused why there are 126 images but use only one surface to project image, is it to improve model's generalization performance?

**Comment by BingyaoHuang** [comment]
@whattoshow cam/warp/ref are used by TPS method only, our CompenNet does not use them, except the gray image (i.e., the surface image).

**Comment by whattoshow** [comment]
@BingyaoHuang The train result of my experiment was good, but I used a simpler stripe surface to project and capture images, the   projector compensation image  looks like a little bit blurry, so I want to do more experiments to check out the blurry problem.

**Comment by BingyaoHuang** [comment]
@whattoshow Sure. For the blurry issue, these discussions (#4, #6) may help.

**Comment by whattoshow** [comment]
![fc28929b-f56d-4056-b234-ad496ed994b4](https://user-images.githubusercontent.com/31459354/137670458-11b330c3-ec0a-4a7d-8788-0a2c80f1e8e5.jpg)
@BingyaoHuang Hi, I met the same problem as sapoluri 's  in ( #4) , when I used the prj/cmp image to project, I found the results look not good.
Two compensation image projected on a 10 stripe surface: 
![fc28929b-f56d-4056-b234-ad496ed994b4](https://user-images.githubusercontent.com/31459354/137670458-11b330c3-ec0a-4a7d-8788-0a2c80f1e8e5.jpg)
![20901318](https://user-images.githubusercontent.com/31459354/137670359-ddd7f310-2428-4572-bf59-2df40cff0982.jpg)
![194988870](https://user-images.githubusercontent.com/31459354/137670373-00b45326-44b6-4d83-b121-9130485215ca.jpg)
Training curve:
![Valid self_10stripes_CompenNet_l1+ssim_500_64_1000_0 001_0 2_800_0 0001](https://user-images.githubusercontent.com/31459354/137670408-ea6ad413-68a1-48ae-b3dc-9662e3daaa48.jpg)

**Comment by BingyaoHuang** [comment]
> did you use the 256*256 images from the benchmark dataset?
Yes, my projector was set to 800x600, I displayed 256x256 image and made it full screen, so the display will automatically resize the 256x256 window to 600x600 (keep aspect ratio).

> the camera captured images are all with mosaic of the projector's DMD lens
Yes, I can see the mosaic in your camera-captured images. Can you see the mosaic with your bear eyes? If so, maybe the camera is too close to the surface? What is your camera resolution?
